verification/VerifyMetrics.wdl

```diff
@@ -74,16 +83,48 @@ task CompareMetricFiles {
       --INPUT ~{file1} \
       --INPUT ~{file2} \
       --OUTPUT ~{output_file} \
-      ~{true="--METRICS_TO_IGNORE" false="" length(metrics_to_ignore) > 0} ~{default="" sep=" --METRICS_TO_IGNORE " metrics_to_ignore}
+      ~{true="--METRICS_TO_IGNORE" false="" length(metrics_to_ignore) > 0} ~{default="" sep=" --METRICS_TO_IGNORE " metrics_to_ignore};
```

Comment:
do we need the semicolon at the end?

Reply:
Not really. It's an oversight on my part, but it allows for additional commands on that block. Originally I was doing the collection of error messages in this task, after the semicolon. But later on decided to write a separate task for it. I'm hesitant to remove it though cause it will trigger another round of tests and approval but won't affect any functionality. Do you prefer I remove it?

Reply:
nope! fine with me to leave!